README.rst

```diff
@@ -4,11 +4,11 @@
 .. image:: https://github.com/scipy-lectures/scipy-lecture-notes/workflows/test/badge.svg?branch=main
   :target: https://github.com/scipy-lectures/scipy-lecture-notes/actions?query=workflow%3A%22test%22
 
-===================
-Scipy-Lecture-Notes
-===================
+===============================
+Scientific-Python-Lecture-Notes
+===============================
 
-This repository gathers some lecture notes on the scientific Python
+This repository gathers some lecture notes on the Scientific Python
```

Comment:
```suggestion
This repository gathers some lecture notes on the scientific Python
```
We might not want to capitalize "scientific" in some places if we want to emphasize that "Scientific Python" does not claim the ecosystem.